### PR TITLE
Fix button color on related conent pane.

### PR DESCRIPTION
**Pull request by justinlittman**
image

### CODE REVIEW

File: app/components/elements/forms/nested_component.html.erb [hunk 1]
```diff
@@ -2,6 +2,11 @@
   <legend class="h5">
     <%= header_label %>
   </legend>
+  <% delete_button = capture do %>
```
Comment (justinlittman) [inline app/components/elements/forms/nested_component.html.erb:5]: Why DRY this? I initially missed that there were 2 identical copies of the button.

File: app/components/elements/forms/nested_component.html.erb [hunk 2]
```diff
@@ -2,6 +2,11 @@
...
+    <%= render Elements::ButtonComponent.new(classes: 'border border-0 float-end', variant: :'outline-primary', size: :lg, data: { action: 'click->nested-form#delete' }) do %>
+      <i class="bi bi-trash"></i>
```
Comment (mjgiarlo) [inline app/components/elements/forms/nested_component.html.erb:7]:
```suggestion
      <%= helpers.delete_icon %>
```

File: app/components/elements/forms/nested_component.html.erb (hunk 1 above)
Comment (justinlittman) [reply app/components/elements/forms/nested_component.html.erb]: I un-DRYed (wet?) it and added a comment to help clarify.

**Review by mjgiarlo — APPROVED** [review]
Appreciate the comment, and the CSS fix!

**Pull request merged**